eor_limits/plot_eor_limits.py

```diff
@@ -127,8 +129,14 @@ def make_plot(
     linewidths=None,
     bold_papers=None,
     fontsize=15,
+    markersize=150,
+    fig_ratio=None,
+    sensitivities: Optional[dict] = None,
+    sensitivity_style: Optional[dict] = None,
```

Comment:
Please add these to the docstring, I don't know what they do.

Reply:
OK, added them